composer.json

```diff
@@ -17,7 +17,9 @@
         "symfony/polyfill-php70": "^1.3",
         "symfony/security": "^2.8|^3.0",
         "symfony/options-resolver": "^2.8|^3.0",
-        "ircmaxell/random-lib": "~1.0"
+        "ircmaxell/random-lib": "~1.0",
+        "symfony/dependency-injection": "^3.3",
```

Comment:
From what I could tell these should have been required before as I didn't get them installed on a fresh `composer install`

Reply:
`^3.3` - will we need to add `^2.7|^3.3` instead?

Reply:
Ah good spot, I'll add that in